zerver/webhooks/facebook/doc.md

```diff
@@ -0,0 +1,74 @@
+{!create-stream.md!}
+
+Next, on your {{ settings_html|safe }},
+[create a bot](/help/add-a-bot-or-integration) for
+{{ integration_display_name }}. Make sure that you select
+**Incoming webhook** as the **Bot type**:
+
+![](/static/images/help/bot_types.png)
+
+The API key for an incoming webhook bot cannot be used to read messages out
+of Zulip. Thus, using an incoming webhook bot lowers the security risk of
+exposing the bot's API key to a third-party service.
+
+Construct the URL for the {{ integration_display_name }}
+bot using the bot's API key and the desired stream name:
+
+`{{ api_url }}{{ integration_url }}?api_key=abcdefgh&stream={{ recommended_stream_name }}&token=sampletoken`
+
+Modify the parameters of the URL above, where `api_key` is the API key
+of your Zulip bot, and `stream` is the stream name you want the
+notifications sent to.  
```

Comment:
Why isn't this using the macro `create-bot-construct-url.md`?  It looks like the only different is the URL; it seems like we could address that without the duplication by just adding a new variable to `webhook-url.md` so it looks like this: 
`{{ api_url }}{{ integration_url }}?api_key=abcdefgh&stream={{ recommended_stream_name }}{{ custom_params }}`    .

Also, I don't understand the `token` -- should it be a secure, randomly generated string, a label for what the integration is that will appear in the Facebook UI, or something else?  "favorite character" string doesn't seem like clear advice for this sort of professional context.

Reply:
Sorry, I can't understand markdown macro even if I read this page(https://chat.zulip.org/api/integration-docs-guide#markdown-macros).

Would you please tell me a bit more specific procedure if possible?

Reply:
@timabbott: Agreed about the macro, I noticed that too but the only reason I didn't suggest that was because adding support for custom params in the webhook URL macro is out of the scope of this PR and should probably be a separate PR altogether. It would most likely involve setting integration-specific context values in the `Integration` instance or something. Not to mention, such a change would apply to all integrations, not just this one! So, it seems like a project of its own!

@nyan-salmon: I agree that the `token` string should be explained in more detail. Could you please investigate further and see if Facebook's docs have anything on that?

Reply:
@eeshangarg I think it's reasonable to leave this for a follow-up issue.